examples/simple_tracking.cpp

```diff
@@ -112,6 +112,7 @@ int main(int argc, char *argv[]) {
   MultiTracker<Filter, 4> mtrk;    // state [x, v_x, y, v_y]
   CVModel cvm(5.0, 5.0);           // CV model with sigma_x = sigma_y = 5.0
   CartesianModel ctm(0.5, 0.5);              // Cartesian observation model
+  CartesianModel3D ctm(0.5, 0.5);              // Cartesian observation model
```

Comment:
This is not ideal, maybe put a comment that people need to decide which one to instantiate.